packages/gatsby-transformer-raml/README.md

```diff
@@ -75,6 +75,8 @@ Example for reading all types:
       enumeration
       examples {
         name
+        displayName
+        description
```

Comment:
@emmenko I guess a change like this is sufficient?

Reply:
Yeah it should work.